.codecov.yaml

```diff
@@ -1,3 +1,11 @@
+coverage:
+  status:
+    project:
+      default:
+        target: 80
+        threshold: 1%
+    patch:
+      default: false
```

Comment:
Can you add a comment explaining what `default: false` does?  and/or link to docs?

Reply:
Done